web/src/components/countryhistorypricesgraph.jsx

```diff
@@ -19,7 +19,8 @@ const prepareGraphData = (historyData) => {
   const valueAxisLabel = `${currencySymbol || '?'} / MWh`;
 
   const priceMaxValue = d3Max(historyData.map((d) => (d.price || {}).value));
```

Comment:
```suggestion
  const priceMaxValue = d3Max(historyData.map((d) => d.price?.value));
```
GitHub wouldn't let me add a suggestion on the other lines for some reason but I think it would be nice if we cleaned up the code a bit here (both the new and existing code)

Optional Chaining should also be used on line 18 and 22.

Reply:
Good points. I'll await review from @Kongkille before making all the changes in one batch if that's fine on your side.